plugins/azure-devops-backend/api-report.md

```diff
@@ -71,6 +72,8 @@ export class AzureDevOpsApi {
     repoName: string,
     top: number,
   ): Promise<RepoBuild[]>;
+  // (undocumented)
+  getTeamMembers(team: Team): Promise<TeamMember[] | undefined>;
```

Comment:
This is the usage:
![image](https://user-images.githubusercontent.com/55280588/148028517-6c8f1b3a-6323-49d7-b40a-73535181df1b.png)

Do you think I should just pass `projectId` and `teamId` and ditch the logging with the team name?

Reply:
Ultimately it's up to you, but the caller of this function now needs to invent extra data that they may not even have at hand. Imagine that you add a profile picture to the Team. Now calling this method gets pretty awkward if all you wanted to do was to just get some members, given a team id :) If it were a private method it'd be one thing, but this one is public.

Reply:
Technically every property of the team is optional, so it's a little unfortunate that you don't know what is required without looking at the function itself. I'd probably vote for `getTeamMembers({ projectId, teamId }: { projectId: string, teamId: string })` and only use `Team` as an output type rather than input types. Then you can remove the check from the `getTeamMembers` as it should always be set. 👍

Reply:
Thanks for the suggestion. I went with `getTeamMembers({ projectId, teamId }: { projectId: string, teamId: string })`.